cmd/schema_gen.go

```diff
@@ -198,3 +269,61 @@ func removeOutdatedColumnsFromSchema(schemaModel *properties.Model, bqColumns []
 	schemaModel.Columns = columnsKept
 	fmt.Println("➖ Columns removed from Schema (no longer in BQ table):", columnsRemoved)
 }
+
+func suggestDocs(file *fs.File, allDocFiles []string) (string, []string) {
+	var modelSuggestions []string
+
+	for ind, col := range file.Schema.Columns {
+		if col.Description == "" {
+			if contains(allDocFiles, col.Name) {
```

Comment:
If I'm reading this right it says if I have a column called "user" and I have a document file called "user.md" then this is going to suggest that it sets the description of that column to the content of that document file?



Can we convert `allDocFiles` to `map[string]interface{}` so can change this line to;

```suggestion
			if _, found := allDocFiles[col.Name]; found {
```

Which would provide a performance boost

Reply:
Nice this makes sense have done this ✅